target/linux/realtek/files-5.10/drivers/clk/realtek/clk-rtl83xx.h

```diff
@@ -14,18 +14,29 @@
 #define RTL838X_PLL_GLB_CTRL			(0x0fc0)
 #define RTL838X_PLL_CPU_CTRL0			(0x0fc4)
 #define RTL838X_PLL_CPU_CTRL1			(0x0fc8)
+#define RTL838X_PLL_CPU_MISC_CTRL		(0x0fcc)
```

Comment:
I don't see the `*_MISC_CTRL` registers being used here. I suppose it doesn't hurt to have the defined, but they are sort of sneaked in with another change.